docs/sc_FunctionalRequirements.csv

```diff
@@ -8,114 +8,114 @@ d)       ATS Command Error Counter
+SC3003.3,SC3003.3,"If neither of the two ATS tables are currently executing, SC shall  reject the Jump command.",
+SC3003.4,SC3003.4,"If multiple commands exist that satisfy the JUMP condition, the commands shall be executed in ascending command number order (as they exist in the ATS table).","If multiple commands exist with the same time as specified in the JUMP command, execute the first command in the table at that time."
+SC3004,SC3004,"Upon receipt of an Enable Continuation Execution of ATS On Error Command, SC shall set the Continuation Execution of ATS On Error Flag to ENABLED.","Provides control on how serious a data integrity check failure is – in this case, continue to execute the ATS."
+SC3005,SC3005,"Upon receipt of a Disable Continuation Execution of ATS On Error Command, SC shall set the Continuation Execution of ATS On Error Flag to DISABLED.","Provides control on how serious a data integrity check failure is – in this case, terminate the current ATS."
 SC4000,SC4000,"Upon receipt of a Start RTS Command, SC shall execute the command-specified RTS, or range of RTS, provided all of the following conditions are met:
 a) The command-specified RTS, or range of RTS, is not currently executing
 b) The RTS, or range of RTS, table(s) is Enabled 
-c) The RTS, or range of RTS, table(s) has been Loaded",Method for starting an RTS.
+c) The RTS, or range of RTS, table(s) has been Loaded",Method for starting an RTS.
 SC4000.1,SC4000.1,"If the conditions are met, SC shall issue an event message indicating the RTS started if the RTS number is less than <PLATFORM_DEFINED> RTS number.","Provides the ground an indication that an RTS started.
-Having a config parameter allows a range of RTSs to be specified such that no event message is sent."
-SC4000.2,SC4000.2,"If the conditions are not met, SC shall reject the command and send an event message.",Ground needs to know if an RTS was not executed.
-SC4001,SC4001,"SC shall dispatch commands within the RTS table, in position order, as the relative time-tag specified in the RTS command expires.",RTS contain relative times between commands.
-SC4001.1,SC4001.1,The time-tag shall be interpreted as the number of seconds to delay relative to the previous RTS command dispatched from that RTS table.,
-SC4001.2,SC4001.2,"For the first command in an RTS table, the delay time shall be relative to the receipt of the RTS Start Command.",Provides the ability to delay the executing of the first command.
+Having a config parameter allows a range of RTSs to be specified such that no event message is sent."
+SC4000.2,SC4000.2,"If the conditions are not met, SC shall reject the command and send an event message.",Ground needs to know if an RTS was not executed.
+SC4001,SC4001,"SC shall dispatch commands within the RTS table, in position order, as the relative wakeup count specified in the RTS command expires.",RTS contain relative wakeup counts between commands.

```

Comment:
I would say "wakeup count... elapses" instead of expires, counts don't really expire.